### ISSUE TITLE
Malformed date from apps

### ISSUE THREAD

**Issue by aescobarr** [Mosquito-Alert/Mosquito-Alert-Mobile-App#12]
TL;DR:

We need to change this kind of datetime constructors:

https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/blob/c3ef283da54cd6afec9d38de75b67cb36cbdf4fd/lib/utils/Utils.dart#L188

From 

DateTime.now().toIso8601String()

to 

DateTime.now().toUtc().toIso8601String()

The first format DateTime.now().toIso8601String() is **NOT UTC**, and it should be. When this kind of date is uploaded to the server, the server assumes this is UTC, so it adds two hours to the time.

You can play around with this at this url: [https://dartpad.dev](https://dartpad.dev

I'm writing this issue at around 16'11 local time. If I execute this code:

```
void main() {    
  print(DateTime.now().toIso8601String());
  print(DateTime.now().toUtc());
  print(DateTime.now().toUtc().toIso8601String());
}
```

It gives as a result
```
2023-06-08T16:11:51.503
2023-06-08 14:11:51.504Z
2023-06-08T14:11:51.504Z
```
We are in UTC zone +2 (+1 for zone, +1 for summer time) so the correct times are the second and third lines. Since dribba is uploading the time in the first format, the server accepts it and assumes it's utc-correct (it's not), so it adds +2 to the time.

As a result, the timestamps for phone_upload_time, creation_time and version_time are incorrect (most of the times are 2 hours in the future of the correct time).
